### PR TITLE
CI integration scripts

### PR DESCRIPTION
**Pull request by laurentheirendt**
This PR adds the following:

- updated `.gitmodules`
- updated `.gitignore`
- first version of `.travis.yml` file
- revised `README` file with badges for build status
- removed `docs`and `m2html`
- test launch scripts (`.sh` and `.m` scripts)
- first test of `readSBML.m`

### CODE REVIEW

**Comment by syarra** [comment]
Thanks @cobrabot  and @laurentheirendt!
We did an amazing job setting up this CI

**Comment by laurentheirendt** [comment]
Thanks @syarra!

File: test/testSBML/testReadSBML.m [hunk 1]
```diff
@@ -4,50 +4,54 @@
...
 
 % set the solver
 %changeCobraSolver('tomlab_cplex');
-changeCobraSolver('cplex_direct');
+solverOK = changeCobraSolver('cplex_direct');
 
-% set the tolerance
-tol = 1e-9;
+if solverOK ~= 1
+    error('Solver cannot be set properly.\n');
```
Comment (syarra) [inline test/testSBML/testReadSBML.m]: Could you comment on why this doesn't work with MATLAB R2104b and R215b?

File: test/testSBML/testReadSBML.m [hunk 2]
```diff
@@ -3,9 +3,7 @@
 % Note:
 % the solver libraries must be included separately
 
-%CBTDIR
-
-%addpath(genpath(CBTDIR))
+addpath(genpath([pwd '/../../']))
```
Comment (syarra) [inline test/testSBML/testReadSBML.m]: Could you make it OS independent?

**Pull request merged**